python/lsst/multiprofit/fit_psf.py

```diff
@@ -506,6 +512,7 @@ def fit(
         # dummy size for first iteration
         size, size_new = 0, 0
         fitInputs = FitInputsDummy()
+        catexp_get_psf_image = catexp.get_psf_image
```

Comment:
It's a probably-irrelevant micro-optimization. I could do the same with setting `modeller=self.modeller` to avoid more attribute retrievals in the for loop, for example.